nixos/modules/hardware/video/backlight/mba6x_bl.nix

```diff
@@ -0,0 +1,27 @@
+{ config, lib, ... }:
+
+with lib;
+
+let cfg = config.services.mba6x_bl;
+    kernel = config.boot.kernelPackages;
+    modprobe = "${config.system.sbin.modprobe}/sbin/modprobe";
+in
+{
+  options = {
+    services.mba6x_bl = {
+      enable = mkEnableOption "the mba6x_bl backlight driver for MacBook Air 6,1 and 6,2";
+    };
+  };
+
+  config = mkIf cfg.enable {
+    boot.extraModulePackages = [ kernel.mba6x_bl ];
+    systemd.services.mba6x_blWorkaround = {
+      description = "Remove and reload mba6x to workaround no brightness bug (#43)";
+      wantedBy = [ "multi-user.target" ];
+      serviceConfig = {
+        Type = "oneshot";
+        ExecStart = "/bin/sh -c \"${modprobe} -r mba6x_bl && ${modprobe} mba6x_bl\"";
```

Comment:
Please use `script` instead of calling `sh` yourself. It's documented over here: https://nixos.org/nixos/manual/options.html#opt-systemd.services._name_.script